### ISSUE TITLE
Context path and reverse proxy (as traefik)

### ISSUE THREAD

**Issue by lucaspouzac** [swarmpit/swarmpit#413]
Hi, is it possible to configure swampit behind a reverse proxy with a context path (#326) ?

Maybe, you need use x-forwarded-prefix header to add path-prefix ? https://docs.traefik.io/v1.7/basics/#matchers

```
app:
    image: swarmpit/swarmpit:1.8
      deploy:
        labels:
          traefik.port: 8080
          traefik.frontend.rule: "Host:${DOMAIN};PathPrefixStrip:/swarmpit/"
```

Actually, I get 404 because index.html assumes that css and js folders are located in the root folder.

![image](https://user-images.githubusercontent.com/758899/69147726-0070ef80-0ad3-11ea-92d9-612e87836c43.png)

**Comment by lucaspouzac** [comment]
in reference #326

**Comment by lumir-mrkva** [comment]
@lucaspouzac it should be working just on relative paths without any configuration, let me investigate

**Comment by polyzois** [comment]
@lucaspouzac have you tried using PathPrefix rather than PathPrefixStrip? Strip will remove the context before forwarding. So swarmpit will believe you called it with "/"

**Comment by lucaspouzac** [comment]
I not try, but If I keep the context-path, my container doesn't have this context path so I should get a 404 on the index.html. 

I just forget "deploy" ont this example, but it's present on my test.

**Comment by nohaapav** [comment]
Confirmed. Reverse proxy is broken since introduction of reitit routing framework so (1.8 and above)

**Comment by nohaapav** [comment]
Fix in 1.8 & latest

**Comment by trajano** [comment]
It didn't appear to work for me.   I checked what Headers it sends using the `containous/whoami`
```
X-Forwarded-For: 10.0.0.2, 10.0.3.3, 10.0.2.10
X-Forwarded-Port: 443
X-Forwarded-Prefix: /viz
X-Forwarded-Proto: https
X-Forwarded-Server: c9923c099c31
X-Real-Ip: 10.0.0.2
```
But I am getting a 404 because it appears to not use the `X-Forwarded-Prefix` in the URLs

![image](https://user-images.githubusercontent.com/110627/111860469-12101480-891e-11eb-86b7-d257adea4dee.png)
